mackup/applications/git.cfg

```diff
@@ -3,3 +3,7 @@ name = Git
 
 [configuration_files]
 .gitconfig
+
+[xdg_configuration_files]
+git/config
+git/credentials
```

Comment:
I guess I have just been using it to tell Git to store them in OS X's Keychain.
Ill take it out as I agree with you on not syncing credentials.